tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -168,6 +168,7 @@ Fix Spawner Status
          ${JL_visible} =  JupyterLab Is Visible
          IF  ${JL_visible}==True
             Maybe Close Popup
+            Maybe Open JupyterLab Sidebar   File Browser
```

Comment:
Another option could have been to replace 
  `Click Element  xpath://span[@title="/opt/app-root/src"] `
for the keyword 
`Navigate Home (Root folder) In JupyterLab Sidebar File Browser`
that already includes the `Maybe Open JupyterLab Sidebar   File Browser`

Reply:
it makes sense. Replaced in the last commit. Thanks! I will run a new PR Test job

Reply:
Thanks to you!